authorizer_lambda/src/authorizer_lambda/main.py

```diff
@@ -1,19 +1,61 @@
 from typing import Any
+
+from authorizer_lambda.cognito_adapter import fetch_owner_from_cognito
 from authorizer_lambda.config import Settings, config
 from authorizer_lambda.models import (
     AuthorizationResponse,
     OwnerResponse,
     TokenAuthorizationRequest,
     OwnerRequest,
+    PolicyDocument,
+    PolicyStatement,
+    StatementEffect,
 )
+import jwt
 
 
 def get_owner_uuid(cognito_id: str, settings: Settings) -> str:
     """Retrieve the owner_uuid from the lambda based on the cognito id and return it."""
     request = OwnerRequest(cognito_id=cognito_id)
```

Comment:
If you're not using this line, it could be removed (but you would use it if you would implement fetching it from the lambda)

Reply:
It makes sense to serialize the payload when it comes from an external source, thanks!